src/Stratis.Bitcoin.Features.MemoryPool/Rules/CheckTxTotalOutVsFeeRule.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Features.MemoryPool.Interfaces;
+
+namespace Stratis.Bitcoin.Features.MemoryPool.Rules
+{
+    /// <summary>
+    /// This rule ensures that the transaction's total out is at least equal to the network's minimum relay fee.
+    /// </summary>
+    public sealed class CheckTxTotalOutVsFeeRule : MempoolRule
```

Comment:
We have already a rule to check fees: CheckFeeMempoolRule and the fee has to be checked against the size of the transaction, not the total output (what's the point of that check?)

Actually you are checking the sum of the output, compared to the min fee computed using the size of the transaction.
If you aim to check if an output is a dust, you should compare the value to the minrelayfee computed against the sum of the size of the inputs required to redeem such generated outputs

Anyway I don't think we need this rule as described in my comment